scripts/download_weights.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
```

Comment:
Lets update the README so that users know that they need to run this file now `cog run python download-weights.py` before `cog predict`

Reply:
Goodl point, updated the readme :-)